CHANGELOG.md

```diff
@@ -26,13 +27,50 @@ This release contains the definition and documentation of
 * Application Profiles API v0.3.0
 
 The API definition(s) are based on
-* Commonalities v0.4.0-rc.2
-* Identity and Consent Management v0.2.0-rc.2
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
 
-## Connectivity Insights API v0.4.0-rc.1
+*  **r1.2 release of connectivity insights has the following API definitions:**
 
-0.4.0-rc.1 is the first release of the API. The version is aligned with
-Commonalities 0.4.0-rc.2 and Identity and Consent Management v0.2.0-rc.2.
+    *  **Connectivity Insights API v0.4.0**
```

Comment:
I suggest to adhere to the https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md
It uses `api-name` so _connectivity-insights_ instead of _Connectivity Insights API_ and so on.
There should be separate sections for each API as in r1.1

Reply:
issue #100 created to track this